src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="khushbu" id="200701600000">
+
+        <update tableName="property_types">
+            <column name="color_picker_default_value" value='787878' />
```

Comment:
I am not sure single quotes is a valid xml syntax, might be worth changing to double quotes